ckanext/switzerland/tests/test_plugin.py

```diff
@@ -16,9 +23,19 @@ def setUp(self):
     def tearDown(self):
         return
 
+    def _load_test_format_mapping(self):
+        try:
+            with open(os.path.join(__location__, 'test_mapping.yaml'),
+                      'r') as format_mapping_file:
+                return yaml.safe_load(format_mapping_file)
+        except (IOError, yaml.YAMLError) as exc:
+            raise exc
+
```

Comment:
this try/except is not needed: you catch the exception and raise it again unchanged. Just remove it :wink: